src/System.Runtime.InteropServices.RuntimeInformation/src/RuntimeInformation.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Runtime.InteropServices
+{
+    public static partial class RuntimeInformation
+    {
+        private const string RuntimeName = ".NET Core";
```

Comment:
.NET Core is a sort of generic umbrella term, though. We've used it to refer to the ".NET Core BCL", which includes the shared and non-shared parts of .NET Native. It's a bit of a vague term in the first place, I don't know if we should use it here.